config/mondo.yml

```diff
@@ -12,7 +12,7 @@ term_browser: ontobee
 
 entries:
 - prefix: /subsets/
-  replacement: https://github.com/monarch-initiative/mondo/releases/download/current/
+  replacement: https://github.com/monarch-initiative/mondo/releases/latest/download/
```

Comment:
this link does not work for me :(

this works:
https://github.com/monarch-initiative/mondo/releases/latest/

Reply:
Wait these links should not do anything.. Try the purls! The full ones